webdriverio/web/webdriver/test/configs/wdio.saucelabs.android.conf.js

```diff
@@ -0,0 +1,67 @@
+const {config} = require('./wdio.shared.conf');
+const build = `Android Chrome WebdriverIO-V6 build-${new Date().getTime()}`
+
+// =========================
+// Sauce Simulator specific config
+// =========================
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY;
+// If you run your tests on Sauce Labs you can specify the region you want to run your tests
+// in via the `region` property. Available short handles for regions are `us` (default) and `eu`.
+// These regions are used for the Sauce Labs VM cloud and the Sauce Labs Real Device Cloud.
+// If you don't provide the region, it defaults to `us`.
+config.region = process.env.REGION || 'us';
+
+// ============
+// Capabilities
+// ============
+config.capabilities = [
+    /**
+     * Android Chrome
+     */
+    {
+        name: "Android 8.1 Web Test",
```

Comment:
The `name` property will bot be overwritten by WebdriverIO, it will only add it when there is no `name`. This makes is more dynamical because each run, based on a spec, will get the root describe. My suggestion would be to remove it and keep it dynamical so it really reflects what happened during a test

Reply:
I see, I removed all names 👍